card/mod/core/set/all/collection.rb

```diff
@@ -228,10 +228,11 @@ def edit_fields
   end
 
   def normalized_edit_fields
-    edit_fields.map do |name, options|
-      options ||= Card.quick_fetch(name).name
+    edit_fields.map do |name_or_card, options|
+      next [name_or_card, options || {}] if name_or_card.is_a?(Card)
```

Comment:
it doesn't seem like this should ever be a card.  we can't have fancy objects (like cards) as values of our voo options, and edit_structure is a voo option (as currently implemented)